caluma/caluma_core/jexl.py

```diff
@@ -13,6 +14,16 @@
 log = getLogger(__name__)
 
 
+def mapby(arr, *keys):
+    if not isinstance(arr, list):
+        return None
+
+    return [
+        [obj.get(key) for key in keys] if len(keys) > 1 else obj.get(keys[0])
+        for obj in arr
+    ]
+
+
```

Comment:
Let's keep all the transforms as methods of `JEXL` and use `self.add_transform("mapby", self.mapby)`.

Reply:
Done